pghoard/basebackup/delta.py

```diff
@@ -88,25 +87,16 @@ def progress_callback(progress_step: ProgressStep, progress_data: ProgressMetric
                 persisted_progress = PersistedProgress.read(self.metrics)
                 progress_info = persisted_progress.get(key)
                 tags: dict = {"phase": progress_step.value}
+                self.last_flush_time = time.monotonic()
 
                 if progress_data["handled"] > progress_info.current_progress:
                     progress_info.update(progress_data["handled"])
                     persisted_progress.write(self.metrics)
-                    self.last_flush_time = time.monotonic()
                     self.metrics.gauge("pghoard.seconds_since_backup_progress_stalled", 0, tags=tags)
-                    self.log.info(
-                        "Updated snapshot progress for %s to %d files; elapsed time since last check: %.2f seconds.",
```

Comment:
It confuses operators, and because it doesn't log frequently, we might think it is stuck.  Because of progress_data["handled"] value